jobs/SDE/Assignments/SDE-FE-Assignment.md

```diff
@@ -0,0 +1,74 @@
+### SDE coding challenge
+
+Here you are! We're happy that you're interested in working for Quantum Inventions! 
+To get a better idea of your current skills, we'd like you to complete a coding challenge - build a web app to display and update resources (e.g - users). 
+The actual requirements are listed further down, but here are the general tech requirements:
+- Make sure your app is runnable on Windows, Mac OS X and Linux
+- Do not include any paid software dependencies
+- Include instructions for setting up and running your application
+- You are free to add any open source CSS framework like bootstrap, material ui as per the need
+- You can use API mock service https://reqres.in/, for API needs. Visit the site for more details.
+
+
+### Description 
+
+Develop a web application which can support following functionalities:
+- Build a **Login** page which takes username and password
```

Comment:
There was a point here in parantheses (maybe in another commit). Please remove it. The candidate should be able to do the linking independently.